### PR TITLE
Disable autoscaling tracking on agent for models that are fixed

### PR DESCRIPTION
**Pull request by sakoush**
**What this PR does / why we need it**:
Previously the agent will track autoscaling metrics for all models regardless of whether the user would want to autoscale them. The scheduler will reject later any autoscaling events. This strategy suffers from the issue that there are unnecessary grpc messages moving from agent->scheduler.

This PR disables tracking autoscaling metrics for models that are fixed (i.e. autoscaling is not set on them). Therefore no events will be fired from agent to scheduler. This is done by the scheduler setting a enable autoscaling flag during model load. There is a caveat with this strategy though:
- we cannot set autoscaling later after the model (replica) has been loaded to a particular server and the user would have to create a new version and do a rolling update

This PR also adds a flag `disable-autoscaling` to disable (model) autoscaling service on scheduler cmdline args. This is set by default in local (docker-compose) deployment as it is not possible to add model replicasgiven we have only one server replica (so far)

### CODE REVIEW

**Comment by seldondev** [comment]
[APPROVALNOTIFIER] This PR is **NOT APPROVED**

This pull-request has been approved by:
To complete the [pull request process](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process), please assign 
You can assign the PR to them by writing `/assign ` in a comment when ready.

The full list of commands accepted by this bot can be found [here](https://jenkins-x.io/v3/develop/reference/chatops/?repo=SeldonIO%2Fseldon-core).

<details open>
Needs approval from an approver in each of these files:

- **[OWNERS](https://github.com/SeldonIO/seldon-core/blob/v2/OWNERS)**

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

**Review by cliveseldon — APPROVED** [review]

**Comment by seldondev** [comment]
[APPROVALNOTIFIER] This PR is **NOT APPROVED**

This pull-request has been approved by: *[cliveseldon](https://github.com/SeldonIO/seldon-core/pull/4501#pullrequestreview-1221775255 "Approved")*
To complete the [pull request process](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process), please assign 
You can assign the PR to them by writing `/assign ` in a comment when ready.

The full list of commands accepted by this bot can be found [here](https://jenkins-x.io/v3/develop/reference/chatops/?repo=SeldonIO%2Fseldon-core).

<details open>
Needs approval from an approver in each of these files:

- **[OWNERS](https://github.com/SeldonIO/seldon-core/blob/v2/OWNERS)**

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

**Pull request merged**
